script/release-notes/notes.js

```diff
@@ -298,35 +301,41 @@ const checkCache = async (name, operation) => {
   return response
 }
 
-const getPullRequest = async (number, owner, repo) => {
-  const name = `${owner}-${repo}-pull-${number}`
-  return checkCache(name, async () => {
+// helper function to add some resiliency to volatile GH api endpoints
+async function runRetryable (fn, maxRetries) {
+  let lastError
+  for (let i = 0; i < maxRetries; i++) {
     try {
-      return await octokit.pulls.get({ number, owner, repo })
+      return await fn()
     } catch (error) {
-      // Silently eat 404s.
-      // We can get a bad pull number if someone manually lists
-      // an issue number in PR number notation, e.g. 'fix: foo (#123)'
-      if (error.code !== 404) {
-        throw error
-      }
+      await new Promise((resolve, reject) => setTimeout(resolve, CHECK_INTERVAL))
+      lastError = error
     }
+  }
+  // Silently eat 404s.
+  if (lastError.status !== 404) throw lastError
+}
+
+const getPullRequest = async (number, owner, repo) => {
+  const name = `${owner}-${repo}-pull-${number}`
+  return checkCache(name, async () => {
+    await runRetryable(octokit.pulls.get({
```

Comment:
This needs to be `return`'ed
```suggestion
    return await runRetryable(octokit.pulls.get({
```